organizing_hub/templates/partials/organizing_hub_header.html

```diff
@@ -26,8 +26,12 @@
             {% if not static_content and request.user.is_authenticated %}
             <div class="col-md-6">
               <div class="uppercase text-right">
+                {% local_group as has_group %}
```

Comment:
The convention of `has_*` implies this is a boolean True/False value. Suggest to not use this naming convention for something that isnt a boolean.

Reply:
:+1: